pkg/eventshub/options.go

```diff
@@ -334,6 +334,11 @@ func OIDCCorruptedSignature() EventsHubOption {
 	return compose(envOption(OIDCGenerateCorruptedSignatureTokenEnv, "true"), envOIDCEnabled())
 }
 
+// OIDCSubject sets the name of the OIDC subject to use by the sender. Defaults to "oidc-<eventshub-name>"
```

Comment:
nit: from the code it seems like I don't need to set this value unless I want a specific SA name, but from the comment it seemed like maybe I did need to use this option if I want the OIDC subject to be set
```suggestion
// OIDCSubject sets the name of the OIDC subject to use by the sender. If this option is not set, it defaults to "oidc-<eventshub-name>"
```

Reply:
:+1: updated